### PR TITLE
File tiles new IA

### PR DESCRIPTION
**Pull request by chri2547**
image

### CODE REVIEW

File: _component_design/file-tile.md [hunk 1]
```diff
@@ -0,0 +1,316 @@
...
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+### Style
+
+Use these specifications when you construct a file tile.
+
```
Comment (palindromebart) [inline _component_design/file-tile.md]: visual needs to change. the icon is called "times" so the style redline for the remove X should be icon: times

**Comment by cyrichardson** [comment]
[helix_File Tiles_cyrEdits.docx](https://github.com/rackerlabs/design-system/files/2362570/helix_File.Tiles_cyrEdits.docx)

**Comment by cyrichardson** [comment]
@chri2547 ^^

File: _component_design/file-tile.md [hunk 2]
```diff
@@ -0,0 +1,316 @@
+---
+title: File Tiles
+parent: Content Areas
+layout: component
+category: Components
+usage: >
+  A file tile displays the name, type, and size of an uploaded file. An uploaded file can be downloaded by a user.
+preview-image: preview-images/checkboxes.svg
```
Comment (araiford) [inline _component_design/file-tile.md]: Preview image needs to be created for File Tiles.  There's a sketch file on GDrive which contains all the preview images.

File: _component_design/file-tile.md [hunk 3]
```diff
@@ -0,0 +1,316 @@
...
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## States
+
+A checkbox can exist in a *loading*, *error*, *focus*, *hover*, or *read-only* state.
```
Comment (araiford) [inline _component_design/file-tile.md]: checkbox should be file tile

File: _component_design/file-tile.md [hunk 4]
```diff
@@ -0,0 +1,316 @@
...
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+### Loading
+
+When a file tile loads, use the loading bar to indicate progress.
```
Comment (araiford) [inline _component_design/file-tile.md]: `While the file is being uploaded, use the...`

File: _component_design/file-tile.md [hunk 5]
```diff
@@ -0,0 +1,316 @@
...
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+### Error
+
+An error state can apply to a file tile when the file tile loads.
```
Comment (araiford) [inline _component_design/file-tile.md]: `...when the file fails to upload.`

File: _component_design/file-tile.md [hunk 6]
```diff
@@ -0,0 +1,316 @@
...
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+### Hover
+
+The hover state indicates that the user can download or remove the file.
```
Comment (araiford) [inline _component_design/file-tile.md]: `...user can click to download or...`

File: _component_design/file-tile.md (hunk 3 above)
Comment (chri2547) [reply _component_design/file-tile.md]: done.

File: _component_design/file-tile.md (hunk 4 above)
Comment (chri2547) [reply _component_design/file-tile.md]: done.

File: _component_design/file-tile.md (hunk 5 above)
Comment (chri2547) [reply _component_design/file-tile.md]: done.

File: _component_design/file-tile.md (hunk 6 above)
Comment (chri2547) [reply _component_design/file-tile.md]: done.

File: _component_design/file-tile.md (hunk 1 above)
Comment (araiford) [reply _component_design/file-tile.md]: This was resolved

**Pull request merged**